lighthouse-core/audits/accessibility/duplicate-id-aria.js

```diff
@@ -1,38 +1,36 @@
 /**
- * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
after deleting `duplicate-id.js` git now thinks this was a move + slight change for this file, which is distracting for review but w/e